tests/integration/cli/classification/test_classification.py

```diff
@@ -105,6 +105,14 @@ def test_otx_train(self, template, tmp_dir_path):
         args1["--load-weights"] = f"{template_work_dir}/trained_{template.model_template_id}/weights.pth"
         otx_train_testing(template, tmp_dir_path, otx_dir, args1)
 
+    @e2e_pytest_component
+    @pytest.mark.skipif(TT_STABILITY_TESTS, reason="This is TT_STABILITY_TESTS")
+    @pytest.mark.parametrize("template", templates, ids=templates_ids)
+    def test_otx_multi_gpu_train(self, template, tmp_dir_path):
```

Comment:
We should skip this test in a non-multi-gpu (single gpu or cpu) environment.

Reply:
Thanks for comment. I'll update it.